### DISCUSSION TITLE
Scavenger Hunt Discussion

### DISCUSSION THREAD

**Discussion by BioNCphilic** [earthlab-education/Earth-Analytics-AY24#108] [General]
The scavenger hunt says "Start your own discussion in one of the Community categories" . It looks like the categories are: announcements, general, ideas, Q&A, reading responses and show & tell. I'm thinking a new discussion is appropriate in the general category because I'm assuming the show and tell involves coding and not a picture of my weekend activities.

**Comment by VictoriAragon** [comment]
I'm having difficulty trying to figure out how to create a pull request for this task section. Need a walk through

**Comment by amarangunic** [comment]
Hi VictoriAragon, I am also looking into how to make the pull request.

**Comment by eculler** [comment]
Thanks @BioNCphilic, looks great!

@VictoriAragon @amarangunic for the pull request, I think it's best to ask about that in class or office hours if your GitHub looks different from ours in the videos. Usually there's some kind of notification suggesting that you could make a pull request...but sometimes you have to do it manually and that can be tricky. We'll get lots of practice over the semester, though, so be sure to keep letting us know if you're having trouble with these.

[This article](https://cu-esiil-edu.github.io/esiil-learning-portal/foundations/pages/03-git-github/02-github-collaboration/02-pull-requests.html) from our textbook has some more information in creating pull requests (and also instructions for the home towns activity for this week)
